src/components/Menu/src/MenuButton.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import BaseProps from '@gemeente-denhaag/baseprops';
+import './menubutton.css';
+import { ChevronDownIcon } from '@gemeente-denhaag/icons';
+
+export interface MenuButtonProps extends BaseProps {
+  /**
+   * Determines the url the menu button refers to
+   * Only applicable to the normal variant
+   */
+  href?: string;
+
+  /**
+   * Determines the Onclick function the menu button referst to
+   * Only applicable to the Expandable variant
+   */
+  onclick?: (event: React.MouseEvent<HTMLButtonElement>) => void;
+
+  /**
+   * Determines if the Menu Button is expandable
+   */
+  variant?: 'normal' | 'expandable';
+}
+
+export const MenuButton: React.FC<MenuButtonProps> = (props: MenuButtonProps) => {
+  let component = null;
+  if (props.variant === 'expandable') {
+    component = (
+      <button id={props.id} onClick={props.onclick} className="denhaag-menu-button">
+        {props.children}
+        <span className="denhaag-menu-button__chevron">
+          <ChevronDownIcon aria-label="ChevronDownIcon" />
+        </span>
+      </button>
+    );
+  } else {
+    component = (
+      <a id={props.id} href={props.href} className="denhaag-menu-button">
+        {props.children}
+      </a>
+    );
+  }
```

Comment:
I think it'll be easier to discern the code for each variant (since they're practically different components) if we do something like this (just a quick type-out, the order is prob not optimal)
```jsx
export const MenuButton: React.FC<MenuButtonProps> = (props: MenuButtonProps) => {
  return props.variant === 'normal' ? menuButton(props) : expandableMenuButton(props);
};

const menuButton = (props: MenuButtonProps) => (
  <a id={props.id} href={props.href} className="denhaag-menu-button">
    {props.children}
  </a>
);

const expandableMenuButton = (props: MenuButtonProps) => (
  <button id={props.id} onClick={props.onclick} className="denhaag-menu-button">
    {props.children}
    <span className="denhaag-menu-button__chevron">
      <ChevronDownIcon aria-label="ChevronDownIcon" />
    </span>
  </button>
);

export default MenuButton;
```

Reply:
Shouldn't we have done this with Card/Case too? I don't think it's a bad idea, but I do think we should do these things consistently